src/media/css/site.styl

```diff
@@ -257,6 +257,12 @@ a:active {
     }    
 }
 
+@media $wide-tablet {
+    .subheader.hidden {
```

Comment:
``[data-page-type~=...]`` I am a bit confused about it. I am seeing in different ``data-page-type`` we have used `website` , `homepage`, `detail`, `app-list` etc. But what will i use here ?